bot/discord/cmd_handler.go

```diff
@@ -11,8 +11,24 @@ import (
 	"github.com/desmos-labs/hephaestus/types"
 )
 
+// CmdHandler represents a function that extends a disgord.HandlerMessageCreate to allow it to return an error
+type CmdHandler = func(s disgord.Session, h *disgord.MessageCreate) error
```

Comment:
Move `CmdHandler` from `types` folder since telegram and discord have different handler structure.